packages/pipelines/src/hooks/usePipelineRuns.ts

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { PipelineRunKind } from '../types';
+import { GetNextPage, Selector } from '../types/tekton-results';
+import { PipelineRunModel } from '../models';
+import { useRuns } from './useRuns';
+import { getGroupVersionKindForModel } from '../utils/common-utils';
+import { FetchUtilsType } from '../types/k8s';
+
+export const usePipelineRuns = (
+  fetchUtils: FetchUtilsType,
+  namespace: string,
+  tektonResultsBaseURL: string,
+  isTektonResultEnabled: boolean,
+  options?: {
+    selector?: Selector;
+    limit?: number;
+  },
+  cacheKey?: string,
+): [PipelineRunKind[], boolean, unknown, GetNextPage] =>
+  useRuns<PipelineRunKind>(
+    fetchUtils,
+    getGroupVersionKindForModel(PipelineRunModel),
+    namespace,
+    tektonResultsBaseURL,
+    isTektonResultEnabled,
+    options,
+    cacheKey,
+  );
+
+export const usePipelineRun = (
+  fetchUtils: FetchUtilsType,
+  namespace: string,
+  pipelineRunName: string,
+  tektonResultsBaseURL: string,
+  isTektonResultEnabled: boolean,
```

Comment:
As per my previous comment, we can just pass that config around the tekton-result utilies as well.